local_docker_development/drupal_site_containers.md

```diff
@@ -39,6 +39,17 @@ If you want to connect to a container wherever you are right now with your bash:
 
 *Replace `changeme.com.docker.amazee.io` with the docker container you want to connect to*
 
+### Drush from your host machine
+
+To use drush, you can either connect to the container as above, or add a bash function that will connect for you to run your drush command. To add the function, add this to your .bashrc file:
+
+```
+ddrush() {
+ docker-compose exec --user drupal drupal bash -c "source ~/.bash_envvars && cd /var/www/drupal/public_html/web && drush $@"
```

Comment:
the problem is that not every site has the folder `web` inside the `public_html` folder, it depends how the folder structure is setup...

there is an environment variable `WEBROOT` which contains the folder `web` so we could do something with that to cd into the right folder